### DISCUSSION TITLE
React ClientOnly stucks on fallback in Chrome 53

### DISCUSSION THREAD

**Discussion by teukuridho** [vikejs/vike#1638] [Discussion]
Reproduction:
1. Init vike.js `pnpm create @batijs/app --react --express`
2. Install `vike-react`
3.  Modify Counter component like this 
 ```
export { Page }

import { Counter } from './Counter'
import { ClientOnly } from 'vike-react/ClientOnly';

function Page() {
  return (
    <>
      <h1>Welcome</h1>
      This page is:
      <ul>
        <li>Rendered to HTML.</li>
        <li>
          Interactive.
          <ClientOnly 
            fallback={<p>Loading</p>}
            load={async () => Counter}
          >
            {(Component) => <Component/>}
          </ClientOnly>
        </li>
      </ul>
    </>
  )
}
```
4. Run on chrome 53, ClientOnly stucks on fallback, no errors or warnings in console
![CleanShot 2024-05-11 at 21 53 49@2x](https://github.com/vikejs/vike/assets/51054737/95dd6e6c-bc52-4ec2-a768-f9daa1d02dc5)

how do I resolve this? thank you

**Comment by brillout** [comment]
Chrome 53 was released in 2016, any reason you want to support a browser
version that old?

**Reply by teukuridho** [reply]
yes, I need my app to run on a LG WebOS tv, the latest chrome available for it is chrome 53 sadly, even after firmware upgrade...

**Reply by brillout** [reply]
In general, Vike doesn’t support browsers that old. But feel free to dig and report back your findings; maybe there is a straightforward way to support Chrome 53.
